setup.cfg

```
@@ -97,7 +97,7 @@ filterwarnings =
     ignore:distutils Version classes are deprecated:DeprecationWarning
     ignore:Passing unrecognized arguments to super:DeprecationWarning
     ignore:.*With traitlets 4\.1, metadata should be set using the \.tag\(\) method:DeprecationWarning
-    ignore:`np.bool8` is a deprecated alias for `np.bool_`.  (Deprecated NumPy 1.24):DeprecationWarning
+    ignore:is a deprecated alias for:DeprecationWarning
```

Maybe a little more specific? 😬 
```suggestion
    ignore:.*np\.bool8.*is a deprecated alias for:DeprecationWarning
```